deployment-helper.py

```diff
@@ -0,0 +1,170 @@
+	from tqdm import tqdm
+	logger.info("Downloading release default asset")
+
+	url = "https://api.github.com/repos/sct-data/{}/releases/{}".format(target_repo, release_id)
+	headers = {
+		"Authorization": "token {}".format(GH_TOKEN),
+		"Content-Type": "application/octet-stream",
+	}
+
+	class TqdmUpTo(tqdm):
+		def update_to(self, num_blocks, block_size, total_size):
+			if total_size is not None and self.total is None:
+				self.total = total_size
+			self.update(num_blocks * block_size - self.n)  # will also set self.n = b * bsize
+
+	req = urllib.request.Request(url, headers=headers, method="GET")
+	with urllib.request.urlopen(req) as resp:
+		if resp.getcode() != 200:
+			raise RuntimeError("Bad response: {} / {}".format(resp.getcode(), resp.read()))
+		ret = json.loads(resp.read().decode('utf-8'))
+		logger.info("ret: %s", ret)
+
+		with TqdmUpTo(unit='B', unit_scale=True, unit_divisor=1024, miniters=1, desc='Downloading') as t:
+			urllib.request.urlretrieve(ret['zipball_url'], ret['tag_name']+'.zip', t.update_to)
+			t.total = t.n
+
+def upload_github_asset(target_repo, release_id, asset_path):
+	'''
+	Uploads a release asset to a target release.
+	'''
+	logger.info("Uploading release asset")
+
+	url = "https://uploads.github.com/repos/sct-data/{}/releases/{}/assets?name={}" \
+		.format(target_repo, release_id, asset_path)
+	headers = {
+		"Authorization": "token {}".format(GH_TOKEN),
+		"Content-Type": "application/octet-stream",
+	}
+
+	with io.open(asset_path, "rb") as fin:
+		payload = fin.read()
+	req = urllib.request.Request(url, headers=headers, method="POST", data=payload)
+	with urllib.request.urlopen(req) as resp:
+		if resp.getcode() != 201:
+			raise RuntimeError("Bad response: {} / {}".format(resp.getcode(), resp.read()))
+		ret = json.loads(resp.read().decode('utf-8'))
+		logger.info("ret: %s", ret)
+
+def upload_to_osf(osf_project_id, osf_username, osf_password, asset_path, upload_path):
+	'''
+	Uploads new version of the data to the Open Science Framework.
+	:return: osf download url of the newly uploaded asset.
+	'''
+	from osfclient import OSF
+
+	osf = OSF()
+	osf.login(username=osf_username, password=osf_password)
+
+	project_storage = osf.project(osf_project_id).storage()
+
+	logger.info('Uploading file to OSF project [%s]', osf_project_id)
+	with open(asset_path, 'rb') as fup:
+		try:
+			project_storage.create_file(upload_path, fup)
+		except Exception as e:
+			logger.error('Upload error: %s', e)
+		else:
+			for file in project_storage.files:
+				# compare strings instead of os.path.samefile() because objects are in different file systems.
+				if file.path == upload_path:
+					return file._download_url
+
+def update_release_(target_repo, release_id, osf_url):
+	'''
+	Include link to osf data in the github release
+	'''
+	pass
+
+
+
+def main(args_in=None):
+
+	def parse_args():
+		parser = argparse.ArgumentParser(
+		description="Create a release on one of the repositories within sct-data organization")
```

Comment:
Since this script would be used, we should use the SCT coding standard.
I think you can use `black` to reformat this script.